examples/algob.config-template.js

```diff
@@ -45,7 +45,7 @@ let accounts = mkAccounts([
 
 let defaultCfg = {
   host: "http://localhost",
-  port: 8080,
+  port: 8080, // change to 8081 if using sandbox
```

Comment:
what do you think about keeping the ports same (8080) ?
P.S: User doesn't have to change each config

Reply:
Agreed. I actually wanted to keep the config same, but wasn't able to get the port `8080` working. Can you try one thing on your end, update the `sandbox-docker-componse.yml` file: change port from `8081` to `8080` at all places, and then run `make sandbox-up`. Maybe there's an issue on my system.
The error it gave was `addr already in use`, but killed the processes running on the same port and it still didn't work. I also tried the reverse, (updating the port `8080` in our create-private-net to `8081` and then tried to start the network, but that didn't work as well). That's why finally i had to use different ports for `sandbox`.

Reply:
ok, checking
I checked once `sandbox-up` takes some time

Reply:
![sock](https://user-images.githubusercontent.com/26249903/106927852-945ec380-6738-11eb-987a-6a0e2773b9b3.png)

i am able to reach this step, after closing the network that was already running on 8080.

Reply:
I think the error persists. Also i think the job didn't complete successfully (after indexer - health) i see an error. 
Is the algod client working within sandbox (t port `8080`). You can  enter the docker container using `sandbox-algod` to enter the shell. Look for algod.net there, if it exists, then connection should be successful.

Reply:
let's ask on discord. how to set port to 8080 or if we can do that.

Reply:
Good Point. Will ask. But did it work with you? Did you check the connection?